src/header/launchbar/LaunchbarButton.tsx

```diff
@@ -33,6 +33,7 @@ export type LaunchbarButtonProps = {
   icon: FunctionComponent<unknown>;
   enabled?: boolean;
   isActive?: boolean;
+  isClickable?: boolean | false;
```

Comment:
- All buttons are clickable... `isClickableWhenActive`?
- `false` is a narrower type than a boolean; meaning that the type `boolean | false` is indistinguishable from type `boolean`